### ISSUE TITLE
[Sid-Nav] Aria-label for hamburger menu is "Open-menu" but the control actually toggles the menu open or closed

### ISSUE THREAD

**Issue by jm986** [carbon-design-system/carbon-addons-iot-react#1675]
### What package is this for?

- [X] React
- [] Angular

### To Reproduce

> Please create a reduced test case in CodeSandbox:
> https://codesandbox.io/s/github/IBM/carbon-addons-iot-react/tree/master/examples/codesandbox

### Expected behavior

Aria-label for the component should be more descriptive and clearer about what behavior the control actually performs

### Environment/versions:

- OS: [e.g. MacOS, Windows]
- Browser: [e.g. chrome, safari]
- `carbon-addons-iot-react` version: [e.g. `v2.60.0`]

### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

**Comment by kevinsperrine** [comment]
The `Header` component is ignoring the SideNav state and only passing the `i18n.openMenu` prop as the aria-label.
